src/ray/object_manager/test/object_manager_stress_test.cc

```diff
@@ -45,16 +48,32 @@ int64_t current_time_ms() {
   return ms_since_epoch.count();
 }
 
+class MockGcsClient : public gcs::RedisGcsClient {
```

Comment:
@ffbin Didn't you mention we should use RedisClient directly? Is it still applied?

Reply:
I think we can remove `RedisGcsClient` directly.

Reply:
@kisuke95 Can you address this part?